src/main/java/org/sonar/plugins/checkstyle/CheckstyleProfileExporter.java

```diff
@@ -224,7 +226,7 @@ private static void appendRuleParameters(Writer writer, ActiveRuleWrapper active
     }
 
     private static void appendModuleProperty(Writer writer, String propertyKey,
-            String propertyValue) throws IOException {
+            @Nullable String propertyValue) throws IOException {
```

Comment:
what tool ask you to do this ?
please move to separate commit if not related to 7.9

Reply:
IDEA when updating to JDK 11